Cubical/Homotopy/Group/SuspensionMapPathP.agda

```diff
@@ -0,0 +1,822 @@
+{-# OPTIONS --safe #-}
+{-
+The goal of this file is to prove that the function
+suspMapΩ : Ωⁿ A → Ωⁿ⁺¹ (Susp A), induced by
+the Freudenthal function σ : A → ΩΣA, gets taken to
+the canonical suspension map
+suspMap : (Sⁿ →∙ A) → (Sⁿ⁺¹ →∙ Susp A)
+given some suitable structure preserving equivalences
+Ωⁿ A ≃ (Sⁿ →∙ A).
+
+The idea is to fill the following diagram
```

Comment:
@ecavallo @aljungstrom  Let's discuss this part on Monday as well. It's so complicated formally and I really wish univalence could help us here